BlargBucket/ViewModels/ReadMeViewModel.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ReadMeViewModel.swift
+//  BlargBucket
+//
+//  Created by Nick Smillie on 12/23/14.
+//  Copyright (c) 2014 Luke Deniston. All rights reserved.
+//
+
+import Foundation
+
+class ReadMeViewModel: NSObject {
+	
+	dynamic var readMeMarkDown: String?
+	var readMeAttributed: NSAttributedString?
+	let repo = AppDelegate.sharedInstance().activeRepo
```

Comment:
Would it be better to find a way to pass the repo to the view model, instead of going to the shared instance to get it here? That was how I had planned to do it, but I couldn't figure out how to send it from RepositoryVM.swift (at least, not without refactoring more than I wanted to).